### ISSUE TITLE
We at Sandstorm like your app!

### ISSUE THREAD

**Issue by neynah** [HelloMeteorBook/2015GlobalHackathon#1]
I discovered your Simple Invoice app during the Meteor Distributed Hackathon. We would love to see your app on the [Sandstorm App Market](https://apps.sandstorm.io/) as it's a great user-friendly app and fulfills a need that none of our other apps currently do.

Let me know if you'd consider packaging it for Sandstorm and we can help you with the [process](https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial/) (and even graphics). :)

**Comment by heliostatic** [comment]
Hi Néna, I'd definitely be interested in packaging for the Sandstorm App Market -- and help with graphics and packaging is definitely welcome.

**Comment by neynah** [comment]
Yes! I'm super excited. :) Please give the [tutorial](https://docs.sandstorm.io/en/latest/vagrant-spk/packaging-tutorial-meteor/) a shot and let me know how it goes. If you're having any questions/problems @paulproteus would be more than happy to help.

I can get working on your graphics once you let me know what you might be looking for- if you're not sure I can help with that too.

**Comment by neynah** [comment]
You can also [book a time slot](https://calendar.google.com/calendar/selfsched?sstoken=UUVCMm5sMVBZNkFSfGRlZmF1bHR8MjM0N2IzODA2NzQ5NjIwMmNkYTIyYTFmODk4ZmY2OTE&pli=1) at your convenience for a hangouts call with @paulproteus. He can answer all your questions and then some. :)

**Comment by neynah** [comment]
@heliostatic I'd love to get started on graphics. Should we begin with a logo?
